src/main.css

```diff
@@ -40,3 +40,17 @@ button.pressed {
 	flex-grow: 1;
 	overflow: auto;
 }
+
+.status-code {
+	margin: 0;
+	padding: 5px;
+	color: #fff;
+}
+
+.status-code.success {
+	background: #00C96B;
+}
+
+.status-code.error {
+	background: #D90000;
+}
```

Comment:
Yeah, might be possible. What we need is a watch mechanism, and a less-compiler. I have also been looking at this. Maybe it just works:

https://github.com/electron/electron-compile